### PR TITLE
Update reference card

### PR DESCRIPTION
**Pull request by XVilka**
- Updated existing commands to match command changes
- Added some more common commands/actions
- Removed rarely needed stuff

### CODE REVIEW

File: src/refcard/intro.md [hunk 1]
```diff
@@ -14,69 +14,85 @@ around a binary and getting information about it.
...
 | fx [flagname]       | Show hexdump of flag  |
 | fC [name] [comment] | Set flag comment      |
 
-## Flagspaces
+## Functions and variables
 
-Flags are created into a flagspace, by default none is selected,
-and listing flags will list them all. To display a subset of flags
-you can use the `fs` command to restrict it.
+Functions appear after auto-analysis or after adding them manually, e.g. with `af` command.
```
Comment (wargio) [inline src/refcard/intro.md:46]:
```suggestion
Functions appear after auto-analysis or after adding them manually using the `af` command.
```

File: src/refcard/intro.md [hunk 2]
```diff
@@ -14,69 +14,85 @@ around a binary and getting information about it.
...
-| ir      | Display relocations      |
-| iz      | List strings (izz, izzz) |
+| Command | Description                 |
+|:--------|:----------------------------|
+| ii      | Information on imports      |
+| iI      | Info on binary              |
+| ie      | Display entrypoint          |
+| iS      | Display sections            |
+| ir      | Display relocations         |
+| iz      | List strings (izz for more) |
```
Comment (wargio) [inline src/refcard/intro.md:80]: i would split this and say that `iz` is the strings found in the read only sections meanwhile `izz` are all the strings found in the raw binary

File: src/refcard/intro.md [hunk 3]
```diff
@@ -14,69 +14,85 @@ around a binary and getting information about it.
...
-| fs *          | Select all flagspaces |
-| fs [space]    | Select one flagspace  |
+| Command             | Description               |
+|:--------------------|:--------------------------|
+| af                  | Analyze function          |
+| af- name            | Remove function "name"    |
+| afb                 | List basic blocks         |
+| afi                 | Show function information |
+| afs                 | Show function signature   |
+| afvl                | Show function variables   |
```
Comment (wargio) [inline src/refcard/intro.md:55]: maybe is worth mentioning agf for the function block graph

File: src/refcard/intro.md [hunk 4]
```diff
@@ -14,69 +14,85 @@ around a binary and getting information about it.
...
 | Command                 | Description                             |
 |:------------------------|:----------------------------------------|
+| help                    | First introduction                      |
 | s (tab)                 | Seek to a different place               |
+| aa[a]                   | Auto analyze (three `a` for more)       |
+| afl                     | List functions                          |
+| afvl                    | List function local vars and args       |
+| avg                     | List globals                            |
+| t                       | List types                              |
+| iz[z]                   | List strings (two `z` for more)         |
```
Comment (wargio) [inline src/refcard/intro.md:24]: this is mentioned twice.

File: src/refcard/intro.md (hunk 2 above)
Comment (XVilka) [reply src/refcard/intro.md]: Removed this occurrence then

File: src/refcard/intro.md (hunk 3 above)
Comment (XVilka) [reply src/refcard/intro.md:55]: Added

**Review by wargio — APPROVED** [review]

**Pull request merged**